server/stycoon/game.go

```diff
@@ -2,20 +2,250 @@ package stycoon
 
 import (
 	"database/sql"
+	"encoding/json"
+	"fmt"
+	"strconv"
 	"time"
 )
 
 type Game struct {
 	Tick     CurrentTick
 	db       *sql.DB
 	lastTick time.Time
+	players  map[string]PlayersValue
 }
 
 func NewGame(db *sql.DB) *Game {
 	game := Game{
 		db:       db,
 		lastTick: time.Now(),
 	}
-	game.getGameTick()
+	game.setGameTick()
 	return &game
 }
+
+func (game *Game) GetData(playerId *int64) (Data, error) {
+	data := Data{
+		CurrentTick: game.Tick,
+		PlayerId:    playerId,
```

Comment:
should this be set if it's nil? Probably does not matter, because it will be omitted while marshaling, right?

Reply:
yes, it is correct like this - missing when user is not logged i